ironfish/src/rpc/routes/wallet/importAccount.ts

```diff
@@ -22,6 +22,7 @@ export const ImportAccountRequestSchema: yup.ObjectSchema<ImportAccountRequest>
       .object({
         name: yup.string().defined(),
         spendingKey: yup.string().defined(),
+        version: yup.number().defined(),
```

Comment:
The version is possibly undefined from old account exports, right?

Reply:
Right, so we add it in the CLI if none is defined.

Reply:
Ahh cool sounds good